### PR TITLE
Docker local development setup

### PR DESCRIPTION
**Pull request by bry-c**
@mingrammer 
I know you will work on this in the future, but maybe this could help others to contribute faster. 🙇

### CODE REVIEW

File: docker/dev/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+diagrams
+black
```
Comment (mingrammer) [inline docker/dev/requirements.txt:2]: I don't think that we have to install the diagrams itself for local development. And we can install the black in the Dockerfile directly with `pip install black` :)

File: docker/dev/setup.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Docker local development setup.
```
Comment (mingrammer) [inline docker/dev/setup.txt:1]: I prefer putting the development documents under the root directory named `DEVELOPMENT.md` (and please use the markdown for any documentations)

File: docker/dev/requirements.txt (hunk 1 above)
Comment (bry-c) [reply docker/dev/requirements.txt:2]: You are right. 😅 I will change it, thanks!

File: docker/dev/setup.txt (hunk 2 above)
Comment (bry-c) [reply docker/dev/setup.txt:1]: If it’s ok, I will move it in that file and covert it to markdown format.

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Development Guide
+
+## Docker local development setup
+
+You should have docker installed in your system, if not click [here](https://docs.docker.com/get-docker/).
+
+1. Go to diagrams root directory.
```
Comment (mingrammer) [inline DEVELOPMENT.md:7]: All sequence number is 1. Please use 1,2,3 .. sequences.

File: docker/dev/setup.txt (hunk 2 above)
Comment (mingrammer) [reply docker/dev/setup.txt:1]: You can delete this file :)

Comment (bry-c) [reply docker/dev/setup.txt]: I forgot to delete 😅

**Pull request merged**